### ISSUE TITLE
image openshift/go-14-centos7:latest not found

### ISSUE THREAD

**Issue by php-coder** [openshift-s2i/s2i-go#10]
Quote from the `README`:

> Alternatively, you can pull the CentOS image from Docker Hub via:
> 
> $ docker pull openshift/go-14-centos7

But when I'm trying to pull the image I got the error:

``` console
$ docker pull openshift/go-14-centos7
Using default tag: latest
Trying to pull repository docker.io/openshift/go-14-centos7 ... not found
Error: image openshift/go-14-centos7:latest not found
```

**Comment by jawnsy** [comment]
Hey!

Thanks for your interest in using this image. As this isn't an "official" Red Hat-supported image (rather it's a community-supported one), I wasn't sure where to push it. I apologize, the docs are incorrect, but I will discuss with others and figure out where to put it.

In the meantime, you can build the image using "docker build" and then manually push to your internal docker registry.

Jonathan

**Comment by jawnsy** [comment]
@bparees What Docker Hub org name should we use for community images like this one?

**Comment by bparees** [comment]
discussed on IRC, answer for now is none, we are still discussing if we want to create a community dockerhub org for images like this and how it will be maintained.

**Comment by tnozicka** [comment]
@bparees @jawnsy any progress on this? It has been a while.

Having those images available would help community a lot to start using S2I. If they have to clone it, build it and maintain it them selves, S2I looses on one of their main points of caring just about application code. People will go for Dockerfile and docker build since it easier and more familiar at that point.

**Comment by jawnsy** [comment]
@tnozicka I agree. There are some ongoing discussions within Red Hat about the intended purpose & future plans of the openshift-s2i organization, and some risks that we have to address before we can move forward.

In the meantime, I could push this under my personal account, as `jawnsy/s2i-go`, if that helps?

**Comment by tnozicka** [comment]
@jawnsy understood

thanks, you don't have to bother with it just because of me, but we should sort it out for the community in the long term. (This needs an automated build somewhere.)

**Comment by 0xmichalis** [comment]
Closing in favor of https://github.com/sclorg/golang-container
